frontend/controller/actions/group.js

```diff
@@ -181,6 +181,7 @@ export default (sbp('sbp/selectors/register', {
           settings: {
             // authorizations: [contracts.CanModifyAuths.dummyAuth()], // TODO: this
             groupName: name,
+            groupCreator: userID,
```

Comment:
I'm not a big fan of creating an ambiguity between usernames and identity contractIDs, and that's what this variable name does.

In various places in the app, a variable like `groupCreator` could refer to the username, or the identity contract ID. I think we should be explicit about when it's an ID instead of a username to avoid confusion.

So for this variable I would choose `groupCreatorID` or `groupCreatorIdentityContractID` instead.

Reply:
This has now been done in general and everything that is an ID has or should have a name indicating so. An exception was made for dictionaries that are indexed by ID. So, for example, `state.profiles[userID]` is used instead of `state.profileIDs[userID]`.